frontend/public/components/edit-yaml.jsx

```diff
@@ -247,6 +249,18 @@ const EditYAMLInner = (props) => {
     [appendYAMLString, allowMultiple, checkEditAccess, t],
   );
 
+  const getResourceKindfromYAML = React.useCallback(
+    (yaml) => {
+      try {
+        const obj = safeLoad(yaml);
+        return getModel(obj)?.id;
```

Comment:
If we are looking for kind of the resource. Can we do `getModel(obj)?.kind;` here instead of id?

Reply:
Updated. Thanks :)